Remotion/Web/Core/Res/Themes/NovaViso/HTML/TabbedMenu.css

```diff
@@ -18,15 +18,12 @@ table.tabbedMenu
 
 table.tabbedMenu tr
 {
+  color: var(--color-standard-text);
+  background-color: var(--color-contrast);
   border-top: var(--tabbedMenu-separator);
   border-bottom: var(--tabbedMenu-separator);
 }
 
-td.tabbedMainMenuCell
-{
-  background-color: var(--color-contrast);
-}
-
 div.tabbedMainMenu div.tabStripTabsPane ul li,
```

Comment:
Just a quick review remark based on code:
color-contrast is gray and was only applied on the mainmenu-row, not the submenu-row. I think this changed the coloring?